readme.md

```diff
@@ -61,7 +61,7 @@ Click the type names for complete docs.
 ### Utilities
 
 - [`Except`](source/except.d.ts) - Create a type from an object type without certain keys. This is a stricter version of [`Omit`](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-5.html#the-omit-helper-type).
-- [`Mutable`](source/mutable.d.ts) - Convert an object with `readonly` keys into a mutable object. The inverse of `Readonly<T>`.
+- [`Mutable`](source/mutable.d.ts) - Convert an object with `readonly` keys into a mutable object. The inverse of `Readonly<T>`. Alternatively creates a type that converts the given keys from `readonly` to mutable and the remaining keys are kept as is, as opposed to converting all keys to mutable.
```

Comment:
This is too long. I think you could just integration a mention of support for only applying to the given keys in the first sentence.

Reply:
Take a look now.